### PR TITLE
Add scraper for varecha.pravda.sk

### PR DESCRIPTION
**Pull request by jose1711**

### CODE REVIEW

File: tests/test_data/varecha.pravda.sk/varechapravdask.json [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    {
+      "ingredients": [
+        "mlieko, 500 ml",
+        "múka hladká, 10 PL",
+        "vajcia , 2 ks celé",
+        "cukor vanilkový, 1 ks bal.",
+        "soľ, štipka"
+      ]
+    }
+  ],
```
Comment (jknndy) [inline tests/test_data/varecha.pravda.sk/varechapravdask.json:25]:
```suggestion
  ],
  "purpose": null

```

**Review by jknndy — CHANGES_REQUESTED** [review]
Hi @jose1711 , thanks for the contribution! 

There are a few changes and additions needed before we can merge;

- [x] [Add a README entry in the support sites section ](https://github.com/hhursev/recipe-scrapers/blob/main/docs/how-to-develop-scraper.md#6-update-the-readme)
- [x] There are a few additional fields that this sites supports that we can include coverage for in the json file. Both `prep_time` & `keywords`
- [x] Address the one code comment

**Comment by jose1711** [comment]
Hope I fixed all issues.

**Review by jknndy — APPROVED** [review]

**Comment by jayaddison** [comment]
This is now available in version `14.58.0` of the library, as published on PyPi.  Thank you!

**Pull request merged**